packages/@aws-cdk/aws-apigateway/test/authorizers/integ.token-authorizer-iam-role.expected.json

```diff
@@ -116,6 +115,7 @@
             "Arn"
           ]
         },
+        "AuthorizerResultTtlInSeconds": 0,
```

Comment:
What's causing this change? 

I believe the default for the value is not zero, hence should be left `undefined`.

Reply:
The default was not specified, I added one for 0, but now realize it should be 5 minutes so it'll now be that (again).